Plate_2_data/4.processing_features/processing_features_plate2_PBMC.sh

```diff
@@ -24,15 +22,18 @@ jupyter nbconvert --to python --output-dir=scripts/ *.ipynb
 echo "Starting processing of plate 2 data"
```

Comment:
Recommend updating code comment above this line to not include extract image features if the python file is not included in the file.